### DISCUSSION TITLE
docs - Requesting info/screenshots for all possible options

### DISCUSSION THREAD

**Discussion by silversword411** [Ylianst/MeshCentral#5799] [Q&A]
Trying to flesh out: https://ylianst.github.io/MeshCentral/meshcentral/devicetabs/

Can you please provide:
screenshots with all possible items that can appear
or 
links to relevant source code areas that give me lists of possible items

![2022-07-02_060441 - mesh all possible options](https://user-images.githubusercontent.com/13395348/176996144-38c5e38c-0c89-4716-a747-1f6d143a8b73.png)

For:

1. All possible headers
2. All possible agent info fields (edit: looks like it's in https://raw.githubusercontent.com/Ylianst/MeshCentral/571a0f1c2d5807d6e3fb9b8c98c7a5c339acf54b/views/default.handlebars around lines 70xx+)
3. all possible buttons (edit: looks like it's in https://raw.githubusercontent.com/Ylianst/MeshCentral/571a0f1c2d5807d6e3fb9b8c98c7a5c339acf54b/views/default.handlebars around lines 711x+)
4. all possible links (edit: looks like it's in https://raw.githubusercontent.com/Ylianst/MeshCentral/571a0f1c2d5807d6e3fb9b8c98c7a5c339acf54b/views/default.handlebars around lines 72xx)

**Comment by si458** [comment]
ill try my best by telling you whats possible, will edit this comment as i go

TABS:
General (stuff about computer),
Desktop (view/control the device like KVM),
Terminal (opens cmd,powershell,ssh to the remote device),
Files (access the devices files, or SFTP to remote device),
Events (stuff thats happened to the device like connects, disconnects, changing settings, remote connections),
Details (information about the device like specs of machine),
Console (allows for JS commands to be issued to the device but also run extra commands from the meshcore (run help to display more options))

MISSING TABS FROM 1:
Intel AMT (information about AMT on the remote device)

2 Information:
Group (which group the device is listed in)
Description (helpful description of remote device like `server running active directory`)
Mesh Agent (self explanitry, shows what OS/architecture you are running)
Operating System (again self explanitry, shows what OS/Version you are running)
Windows Security (only visible on windows devices, shows if AV, Updates, Firewall are `OK`, if not shows `BAD` so they might be an issue)
Active User (shows the currently logged in user)
User Consent (options to notify user/prompt access, etc at the device level)
Notifications (displays web page notifications for device connects/disconnects again at device level)
Tags (sorta like groups, tag devices like `webservers` to search for only `webservers`)

MISSING INFORMATION FROM 2:
Intel AMT (this will show is a device is capable of using AMT and if activated or not)
Antivirus (this will show if it can recognise an installed antivirus and all is `OK` or `BAD`)
Hostname (the devices hostname but can be shown as something else)
![Screenshot 2022-07-02 113413](https://user-images.githubusercontent.com/765314/176997055-b99886e6-dd30-4bde-bebf-26e09c142432.png)

**Comment by silversword411** [comment]
See https://github.com/Ylianst/MeshCentral/pull/4209

So you can see what/how I'm trying to add things.

If you want to add stuff or collaborate live join MeshCentrals Unofficial Discord: https://discord.gg/b58UPsGQvG

**Comment by silversword411** [comment]
> will edit this comment as i go

Before you work too hard, I'm trying to get a base "max options" and then we'll slowly fill the text in for keyword searching with docs. I think @Ylianst will have some test machines with "all the options" for testing. That might be a 1 and done solution
